src/main/app/src/app/shared/edit/edit-select/edit-select.component.ts

```diff
@@ -28,7 +28,7 @@ export class EditVertrouwelijkheidaanduidingComponent extends EditComponent {
 
     init(formField: SelectFormField): void {
         this.option = formField.formControl.value;
```

Comment:
Waar wordt die option eigenlijkj gebruikt?

Reply:
Deze is nodig voor de dropdown zelf, voor als je een andere waarde selecteert en dan op Annuleren drukt. Dan word de oorspronkelijke waarde weer in het veld gezet.